lazysignup/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class LazySignupConfig(AppConfig):
+    name = 'lazysignup'
+    verbose_name = 'Django LazySignup'
```

Comment:
Personally, I think should have done `Lazy Signup` and while CamelCase is not pythonic, Title Case is the convention for the Django admin (Where this would show up)

![screen shot 2015-04-30 at 10 21 14 am](https://cloud.githubusercontent.com/assets/791000/7414757/ad01d0f6-ef22-11e4-869c-7241c88ed202.png)
If you still prefer just `lazysignup` I have no problem with that.